front/app/components/UI/FullscreenModal/index.tsx

```diff
@@ -136,71 +123,10 @@ class FullscreenModal extends PureComponent<Props, State> {
       });
   }
 
-  componentDidUpdate(prevProps: Props) {
-    if (!prevProps.opened && this.props.opened) {
-      this.openModal();
-    } else if (prevProps.opened && !this.props.opened) {
-      this.cleanup();
-    }
-  }
-
   componentWillUnmount() {
     this.subscription?.unsubscribe();
-    this.cleanup();
   }
 
-  openModal = () => {
```

Comment:
The `url` prop was not used, which means this function could not get past the if condition, which means this function is not used. 🙃